velox/expression/tests/CustomTypeTest.cpp

```diff
@@ -109,13 +125,36 @@ class FromFancyIntFunction : public exec::VectorFunction {
                 .build()};
   }
 };
+
+struct FancyIntT {
+  using type = std::shared_ptr<FancyInt>;
+  static constexpr const char* typeName = "fancy_int";
```

Comment:
```
s/static constexpr const char*/static constexpr char*/
```

Reply:
This doesn't compile: ISO C++11 does not allow conversion from string literal to 'char *const'